yt_dlp/extractor/adobepass.py

```diff
@@ -1508,15 +1508,17 @@ def extract_redirect_url(html, url=None, fatal=False):
                     # In general, if you're connecting from a Verizon-assigned IP,
                     # you will not actually pass your credentials.
                     provider_redirect_page, urlh = provider_redirect_page_res
-                    if 'Please wait ...' in provider_redirect_page:
+                    # From non-Verizon IP, still gave 'Please wait', but noticed N==Y; will need to try on Verizon IP
+                    if 'Please wait ...' in provider_redirect_page and "'N'== \"Y\"" not in provider_redirect_page:
```

Comment:
> "'N'== \"Y\""
Are you sure this is not a typo?
`"'N'=='Y'"` would make more sense

Reply:
Yep, not a typo. I could make a more lenient regex, but it seems unchanged for the past 3 years.
![image](https://user-images.githubusercontent.com/482367/130267039-f58aacd5-f5ec-4e32-bd5a-20d787382031.png)

Reply:
```suggestion
                    if 'Please wait ...' in provider_redirect_page and '\'N\'== "Y"' not in provider_redirect_page:
```

Since u need to `\` anyway, might as well stick to conventional single quotes